Parse/src/main/java/com/parse/ParseObject.java

```diff
@@ -4237,7 +4246,26 @@ public void writeToParcel(Parcel dest, int flags) {
 
   /* package */ void writeToParcel(Parcel dest, ParseParcelEncoder encoder) {
     synchronized (mutex) {
-      // Write className and id regardless of state.
+      // Developer warnings.
+      ldsEnabledWhenParceling = Parse.isLocalDatastoreEnabled();
+      boolean saving = hasOutstandingOperations();
+      boolean deleting = isDeleting || isDeletingEventually > 0;
+      if (saving) {
+        Log.w(TAG, "About to parcel a ParseObject while a save / saveEventually operation is " +
```

Comment:
We can do that later.  Would rather have a really big warning msg when this thing is first being used rather than a one-liner so we get some feedback about whether it happens a lot or not a ton.